Library/Homebrew/bottles.rb

```diff
@@ -67,7 +67,7 @@ def self.package(formula_name)
   end
 
   def self.repository(tap=nil)
-    return "bottles" if tap.to_s.empty?
+    return "bottles" if tap.nil? || tap == "Homebrew/homebrew" || tap == "mxcl/master"
```

Comment:
It makes more sense to put `tap.nil? || tap == "Homebrew/homebrew" || tap == "mxcl/master"` as a method.

However, I haven't found a good place to put the method or what's the good name. Part of reason is we have complete different variables all calling tap. It's kinda confusing.